openwisp-utils-qa-checks

```diff
@@ -83,15 +83,18 @@ runisort() {
     || { echoerr "ERROR: Isort check failed!"; FAILURE=1; }
 }
 
+# This check is run when any of the following conditions is true:
+# 1. --force-checkcommit is specified
+# 2. This function is invoked from Travis building a Pull Request
 runcheckcommit() {
-  [ -n "$TRAVIS_PULL_REQUEST" ] || TRAVIS_PULL_REQUEST="false"
-  if [ "$SKIP_CHECKCOMMIT" = "false" -o "$TRAVIS_PULL_REQUEST" != "false" ]; then
+	if [[ $FORCE_CHECKCOMMIT = true ]] ||
+		 [[ $TRAVIS = true && $TRAVIS_PULL_REQUEST = true ]]; then
     echo "Running commit message check"
     checkcommit --message "$COMMIT_MESSAGE" \
     && echo "SUCCESS: Commit name check successful!" \
     || { printf "Checking commit message:\n\n"; printf "%s\n\n" "$COMMIT_MESSAGE"; echoerr "ERROR: Commit name check failed!"; FAILURE=1; }
   else
-    echo "SUCCESS: Commit name check successful!"
+    echo "SKIPPED: Commit name check"
```

Comment:
spotted something odd in the old code, can you replace the instances of "Commit name" with "Commit message" please?